onnx/defs/tensor/old.cc

```diff
@@ -1631,11 +1631,9 @@ output[i_{0}, ..., i_{q-1}, j_{0}, ..., j_{r-2}] = input[j_{0}, k, j_{1}, ..., j
   ]
   axis = 1,
   output = [
-      [
-          [1.0, 1.9],
-          [2.3, 3.9],
-          [4.5, 5.9],
-      ],
+      [[1.0, 1.9]],
```

Comment:
Thanks for the fix. I think the line 1621 also needs to be updated (as in Gather-13)? May be same for Gather-1 also down below?

Reply:
line 1621 is updated as Gather-13 and Gather-1's doc has only input/output definition in example so, I think, no update is needed.